### DISCUSSION TITLE
建议集成一下 OpenAI Whisper 实现语音识别

### DISCUSSION THREAD

**Discussion by matyhtf** [lenML/Speech-AI-Forge#36] [想法 | Ideas]
如题，希望能集成一下 OpenAI Whisper 实现语音识别，一套系统平替 OpenAI Voice API

**Comment by zhzLuke96** [comment]
其实计划中是准备把asr也包括到forge中的

顺带一提，据我初步研究 chattts 本身是有 asr 转写的能力的，和whisper类似，只是没法做翻译任务，不过原作者貌似没说（也有可能 4w 小时的预训练模型效果不行）

**Reply by matyhtf** [reply]
期待
